pool/pool.go

```diff
@@ -271,6 +287,20 @@ func (p *Pool) validateTx(ctx context.Context, tx types.Transaction) error {
 	return nil
 }
 
+func (p *Pool) pollMinSuggestedGasPrice(ctx context.Context) {
+	fromTimestamp := time.Now().UTC().Add(-p.cfg.MinSuggestedGasPriceInterval.Duration)
+	gasPrice, err := p.storage.MinGasPriceSince(ctx, fromTimestamp)
+	if err == state.ErrNotFound {
+		log.Warnf("No suggested min gas price since: %v", fromTimestamp)
+	} else if err != nil {
+		log.Errorf("Error getting min gas price since: %v", fromTimestamp)
+	} else {
+		p.minSuggestedGasPriceMux.Lock()
```

Comment:
Could be it will useful to log (in debug mode) the new suggested gas price. I know it will be retrieved from the gasprice table, but maybe is some situations it will be useful to have it in the logs